### ISSUE TITLE
Research paper to cited

### ISSUE THREAD

**Issue by pathakamarnath** [matt-stam/calcipher#3]
Is there any research work to be cited, if this algorithm is used?

**Comment by mattstam** [comment]
I have not seen anything similar to this algorithm ever get used or researched

**Comment by pathakamarnath** [comment]
No, actually I have to use it for my work. I found it useful for my work. Do I need to cite some paper?

**Comment by mattstam** [comment]
No, but it would be nice if you would credit me or this repo. I'd be curious to know what you're using it for aswell.

**Comment by pathakamarnath** [comment]
Yeah, definitely. Once the work is accepted, I will let you know about it.

**Comment by mattstam** [comment]
@pathakamarnath Still in progress? I'd love to see check it out.
